README.md

```diff
@@ -170,97 +173,116 @@ Tool | Description | Reference
 [htseq-count](http://www-huber.embl.de/HTSeq/doc/count.html) | Tool for counting reads in features | [Anders et al. 2015](https://dx.doi.org/10.1093%2Fbioinformatics%2Fbtu638)
 [Sailfish](http://www.cs.cmu.edu/~ckingsf/software/sailfish/) | Rapid Alignment-free Quantification of Isoform Abundance | [Patro et al. 2014](http://dx.doi.org/10.1038/nbt.2862)
 [Salmon](https://combine-lab.github.io/salmon/) | Fast, accurate and bias-aware transcript quantification | [Patro et al. 2017](http://dx.doi.org/10.1038/nmeth.4197)
+
 <p align="right"><a href="#top">&#x25B2; back to top</a></p>
 
-## NGS - Differential expression analysis
+### Differential expression analysis
 
 Tool | Description | Reference
----- | ----------- | ---------
+--- | --- | ---
 [DESeq2](https://bioconductor.org/packages/release/bioc/html/DESeq2.html) | Differential gene expression analysis based on the negative binomial distribution | [Love et al. 2014](http://doi.org/10.1186/s13059-014-0550-8)
+
 <p align="right"><a href="#top">&#x25B2; back to top</a></p>
 
-## NGS - Utilities
+### Utilities
 
 Tool | Description | Reference
----- | ----------- | ---------
+--- | --- | ---
 [SAMtools](http://samtools.sourceforge.net/) | Utilities for manipulating alignments in the SAM format | [Heng et al. 2009](https://doi.org/10.1093/bioinformatics/btp352)
 [BEDTools](http://bedtools.readthedocs.io/en/latest/) | Utilities for genome arithmetic | [Quinlan and Hall 2010](https://doi.org/10.1093/bioinformatics/btq033)
 [deepTools](https://deeptools.github.io/) | Tools for exploring deep-sequencing data | [Ramirez et al. 2014](https://doi.org/10.1093/nar/gku365), [Ramirez et al. 2016](https://doi.org/10.1093/nar/gkw257)
+
 <p align="right"><a href="#top">&#x25B2; back to top</a></p>
 
 ## Ribosome profiling
 
 Tool | Description | Reference
----- | ----------- | ---------
+--- | --- | ---
 [RiboTaper](https://ohlerlab.mdc-berlin.de/software/RiboTaper_126/) | An analysis pipeline for Ribo-Seq experiments, exploiting the triplet periodicity of ribosomal footprints to call translated regions | [Calviello et al. 2016](https://dx.doi.org/10.1038/nmeth.3688)
+
 <p align="right"><a href="#top">&#x25B2; back to top</a></p>
 
+# Training
+
+To learn about RNA sequencing data analysis, we recommend you to have a look at the training material from the [Galaxy Training network](http://galaxyproject.github.io/training-material/RNA-Seq/), and particularly the tutorial about the [Reference-based RNA-seq data analysis](https://galaxyproject.github.io/training-material//RNA-Seq/tutorials/ref_based).
+
+In the Galaxy RNA workbench, you will also find way to learn about the RNA analyis with some [Galaxy interactive tours](https://github.com/galaxyproject/galaxy-tours) to show you the way into the Galaxy instance, its tools and possibilities.
```

Comment:
In the Galaxy RNA workbench, we also included [Galaxy interactive tours](https://github.com/galaxyproject/galaxy-tours) to guide you through the Galaxy, it's tools and possibilities. ??